.github/workflows/tests_sources.yml

```diff
@@ -45,14 +45,41 @@ jobs:
         export OPENDR_HOME=$PWD
         pip install -r tests/requirements.txt
         python -m unittest discover -s tests
-    - name: Test API
-      if: ${{ matrix.os == 'ubuntu-20.04' && contains(github.event.pull_request.labels.*.name, 'test tools') }}
+  test-tools:
+    needs: cleanup-runs
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'test tools') }}
+    strategy:
+      matrix:
+        package:
+          - engine
+          - utils
+          - perception/activity_recognition
+          - perception/compressive_learning
+          - perception/face_recognition
+          - perception/heart_anomaly_detection
+          - perception/multimodal_human_centric
+          - perception/object_tracking_2d
+          - perception/object_detection_3d
+          - perception/pose_estimation
+          - perception/speech_recognition
+          # - perception/object_tracking_3d
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      if: github.event.pull_request.draft == false
+    - name: Set up Python 3.8
+      if: github.event.pull_request.draft == false
```

Comment:
Not sure it make sense any longer to skip this in case of draft PR.
Does it work if you set the `test tools` label for a draft PR?

Reply:
Removed in commit 77abe5f41c